content/reference/endpoints/bookings.md

```diff
@@ -234,7 +236,7 @@ POST /rentals/:rental_id/bookings
 
 <%= render 'json_response', endpoint: "bookings", request: "create",
   scopes: [
-    { bookings_write_owned: "public-bookings_write_owned" },
+    { bookings_write_owned: "bookings_read-bookings_write" },
```

Comment:
Why this change?

Reply:
It made no sense before. If it has bookings_write_owned scope when you create a booking it returns booking created by app, so it's not limiting attributes, no?

Reply:
True.